frontend/src/metabase/components/ListItem/ListItem.unit.spec.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Route } from "react-router";
+import { screen, getIcon, renderWithProviders } from "__support__/ui";
+import ListItem from "./ListItem";
+
+const ITEM_NAME = "Table Foo";
+const ITEM_DESCRIPTION = "Nice table description.";
+
+function setup({ name, index = 0, ...opts }) {
+  return renderWithProviders(
+    <Route
+      path="/"
+      component={() => <ListItem name={name} index={index} {...opts} />}
+    />,
+    { withRouter: true },
+  );
+}
+
+describe("ListItem", () => {
+  it("should render", () => {
+    setup({
+      name: ITEM_NAME,
+      description: ITEM_DESCRIPTION,
+      icon: "table",
+      url: "/foo",
+    });
+
+    expect(screen.getByText(ITEM_NAME)).toBeInTheDocument();
+    expect(screen.getByText(ITEM_DESCRIPTION)).toBeInTheDocument();
+    expect(getIcon("table")).toBeInTheDocument();
+    expect(screen.getByRole("link")).toHaveProperty(
+      "href",
+      "http://localhost/foo",
+    );
+  });
+
+  it("should render with just the name", () => {
+    setup({ name: ITEM_NAME });
```

Comment:
I could've instantiated [the setup function](https://github.com/metabase/metabase/pull/30635/files#diff-19829cfdb42c57040c5f8b81bf818671183cab1c722ecdc9721cd23778c63850R9) with the default `name` value, but I was wondering would this test loose the readability and the explicitness if it said:
"should render with just the name" and then you see:
```javascript
setup();
expect(screen.getByText(ITEM_NAME)).toBeInTheDocument();

```